src/components/RecommededVideosList.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { useParams } from 'react-router'
+import { useVideos } from '../context'
+import VideoCard from './VideoCard'
+
+function RecommededVideosList() {
+  const {videoId} = useParams()
+  const {videos} = useVideos()
+  const videoPlaying = videos.filter(video => video._id === videoId)
```

Comment:
just a suggestion ,you can try .some() instead .filter as it returning only one video in array

Reply:
Some return true or false.I want that video's data thatswhy filter